### ISSUE TITLE
Eigen/src/ is missing after pip install eigency

### ISSUE THREAD

**Issue by fritzo** [wouterboomsma/eigency#17]
I've been able to install eigency under python 3, but I'm having trouble with python 2.7. Any ideas?
```
$ pip install eigency
Downloading/unpacking eigency
  Downloading eigency-1.75.tar.gz (666kB): 666kB downloaded
  Running setup.py egg_info for package eigency
    Compiling eigency/conversions.pyx because it changed.
    Compiling eigency/core.pyx because it changed.
    [1/2] Cythonizing eigency/conversions.pyx
    [2/2] Cythonizing eigency/core.pyx
    warning: no files found matching 'eigency/*.pyd'
    warning: no previously-included files matching 'CMakeLists.txt' found under directory 'eigency/eigen_3.2.8/Eigen'
Requirement already satisfied (use --upgrade to upgrade): numpy in /Library/Python/2.7/site-packages (from eigency)
Requirement already satisfied (use --upgrade to upgrade): cython in /Users/fritz/.virtualenvs/foo/lib/python2.7/site-packages (from eigency)
Installing collected packages: eigency
  Running setup.py install for eigency
    building 'eigency.conversions' extension
    cc -fno-strict-aliasing -fno-common -dynamic -arch x86_64 -arch i386 -g -Os -pipe -fno-common -fno-strict-aliasing -fwrapv -DENABLE_DTRACE -DMACOSX -DNDEBUG -Wall -Wstrict-prototypes -Wshorten-64-to-32 -DNDEBUG -g -fwrapv -Os -Wall -Wstrict-prototypes -DENABLE_DTRACE -arch x86_64 -arch i386 -pipe -I/Library/Python/2.7/site-packages/numpy/core/include -Ieigency/eigen_3.2.8 -I/System/Library/Frameworks/Python.framework/Versions/2.7/include/python2.7 -c eigency/conversions.cpp -o build/temp.macosx-10.11-intel-2.7/eigency/conversions.o
    c++ -bundle -undefined dynamic_lookup -arch x86_64 -arch i386 -Wl,-F. build/temp.macosx-10.11-intel-2.7/eigency/conversions.o -o build/lib.macosx-10.11-intel-2.7/eigency/conversions.so
    building 'eigency.core' extension
    cc -fno-strict-aliasing -fno-common -dynamic -arch x86_64 -arch i386 -g -Os -pipe -fno-common -fno-strict-aliasing -fwrapv -DENABLE_DTRACE -DMACOSX -DNDEBUG -Wall -Wstrict-prototypes -Wshorten-64-to-32 -DNDEBUG -g -fwrapv -Os -Wall -Wstrict-prototypes -DENABLE_DTRACE -arch x86_64 -arch i386 -pipe -I/Library/Python/2.7/site-packages/numpy/core/include -Ieigency/eigen_3.2.8 -I/System/Library/Frameworks/Python.framework/Versions/2.7/include/python2.7 -c eigency/core.cpp -o build/temp.macosx-10.11-intel-2.7/eigency/core.o
    In file included from eigency/core.cpp:469:
    In file included from eigency/eigency_cpp.h:10:
    eigency/conversions_api.h:182:12: warning: 'static' function 'import_eigency__conversions' declared in header file should be declared 'static inline' [-Wunneeded-internal-declaration]
    static int import_eigency__conversions(void) {
               ^
    1 warning generated.
    In file included from eigency/core.cpp:469:
    In file included from eigency/eigency_cpp.h:10:
    eigency/conversions_api.h:182:12: warning: 'static' function 'import_eigency__conversions' declared in header file should be declared 'static inline' [-Wunneeded-internal-declaration]
    static int import_eigency__conversions(void) {
               ^
    1 warning generated.
    c++ -bundle -undefined dynamic_lookup -arch x86_64 -arch i386 -Wl,-F. build/temp.macosx-10.11-intel-2.7/eigency/core.o -o build/lib.macosx-10.11-intel-2.7/eigency/core.so
    warning: no files found matching 'eigency/*.pyd'
    warning: no previously-included files matching 'CMakeLists.txt' found under directory 'eigency/eigen_3.2.8/Eigen'
  Could not find .egg-info directory in install record for eigency
Successfully installed eigency
Cleaning up...
```
Now my package using eigency fails to build with an error like
```
$ pip install -e $MY_PACKAGE
...
/Users/fritz/.virtualenvs/foo/lib/python2.7/site-packages/eigency/eigen_3.2.8/Eigen/Core:15:10: fatal error: 'src/Core/util/DisableStupidWarnings.h' file not found
```
and indeed the Eigen source is nowhere to be found
```
$ tree $VIRTUAL_ENV/lib/python2.7/site-packages/eigency
/Users/fritz/.virtualenvs/foo/lib/python2.7/site-packages/eigency
├── __init__.py
├── __init__.pyc
├── conversions.h
├── conversions.pxd
├── conversions.pyx
├── conversions.so
├── conversions_api.h
├── core.pxd
├── core.pyx
├── core.so
├── eigen_3.2.8
│   └── Eigen
│       ├── Array
│       ├── Cholesky
│       ├── CholmodSupport
│       ├── Core
│       ├── Dense
│       ├── Eigen
│       ├── Eigen2Support
│       ├── Eigenvalues
│       ├── Geometry
│       ├── Householder
│       ├── IterativeLinearSolvers
│       ├── Jacobi
│       ├── LU
│       ├── LeastSquares
│       ├── MetisSupport
│       ├── OrderingMethods
│       ├── PaStiXSupport
│       ├── PardisoSupport
│       ├── QR
│       ├── QtAlignedMalloc
│       ├── SPQRSupport
│       ├── SVD
│       ├── Sparse
│       ├── SparseCholesky
│       ├── SparseCore
│       ├── SparseLU
│       ├── SparseQR
│       ├── StdDeque
│       ├── StdList
│       ├── StdVector
│       ├── SuperLUSupport
│       └── UmfPackSupport
└── eigency_cpp.h
```
Thanks for a great package!

**Comment by fritzo** [comment]
As a workaround, I've `pip install --editable` eigency.

**Comment by uncommoncode** [comment]
One way to get around this is to set your own Eigen include path in `include_dirs` and use `eigency.get_includes(include_eigen=False)`

**Comment by rmjarvis** [comment]
I think the updates last April to the MANIFEST.in file should have fixed this.  But the current version on pypi is still from 2016.  Any chance you could just re-upload the sdist to pypi?

I have a python module that uses Eigen in the python layer.  I was hoping to use this package to make it pip installable without requiring users to separately install Eigen.  But if `pip eigency` doesn't actually install the Eigen/src directory, then it's not so helpful for me.  Thanks!

**Comment by BTOdell** [comment]
Verified fixed in 1.78 (may have been fixed earlier though).
